config/hubs/pangeo-hubs.cluster.yaml

```diff
@@ -0,0 +1,51 @@
+name: pangeo-hubs
+image_repo: "us-central1-docker.pkg.dev/two-eye-two-see/pilot-hubs-registry/base-user"
+provider: gcp
+gcp:
+  key: secrets/pangeo-hubs.json
+  project: pangeo-181919
+  cluster: pangeo-hubs-cluster
+  zone: us-central1-b
+support:
+  config:
+    grafana:
+      ingress:
+        hosts:
+          - grafana.pangeo.2i2c.cloud
+        tls:
+          - secretName: grafana-tls
+            hosts:
+              - grafana.pangeo.2i2c.cloud
+hubs:
+  - name: geospatial-workshop
+    domain: geospatial-workshop.pangeo.2i2c.cloud
+    template: basehub
+    auth0:
+      connection: ""
+    config:
+      # docs_service:
```

Comment:
Because I just copy-pasted the config from 2i2c-staging and I wasn't sure if it would be needed for the geospatial workshop, but also didn't want to lose the structure in case it _was_ required

Reply:
Let's get rid of this for now, we can always copy paste it back if needed.